lib/modules/orangered.js

```diff
@@ -205,48 +271,112 @@ const setupFaviconBadge = _.once(() => {
 const orangeredElements = () => ({
 	...nativeButtons(),
 	...floatingButtons(),
-	modmailButtonSelector: '#modmail',
 });
 
 const $orangeredElements = () => Object.values(orangeredElements()).reduce(($all, ele) => $all.add(ele), $());
 
-const nativeButtons = _.once(() => {
-	const nativeInboxButton = document.querySelector('#header-bottom-right #mail');
-	let nativeInboxCount = document.querySelector('#header-bottom-right .message-count');
-	if (!nativeInboxCount && (module.options.updateCurrentTab.value || module.options.updateOtherTabs.value)) {
+const nativeButtons = _.once(() => _nativeButtons(document.body, true));
+const _nativeButtons = (container = document.body, addIfMissing) => {
```

Comment:
Super nit: all other (well, except `orangeredElements`) top-level functions are function declarations when they can be, why not this one too?